scripts/common/common.sh

```diff
@@ -385,6 +385,26 @@ function try_1m() {
     done
 }
 
+# try a command every 2 seconds until it succeeds, up to 150 tries max; useful for kubectl commands
+# where the Kubernetes API could be restarting
+function try_5m() {
```

Comment:
this doesn't actually try for 5m - it tries 150 times
if each try takes 6s, then with the 2s wait it would be 20m